app/assets/js/rtMedia.backbone.js

```diff
@@ -405,11 +400,7 @@ jQuery( function( $ ) {
 			}
 
 			change_rtBrowserAddressUrl( href, '' );
-			if ( $( this ).data( 'page-type' ) == 'num' ) {
-				galleryObj.getNext( nextpage, $( this ).parent().parent().parent().parent().parent(), $( this ).parent().parent() );
-			} else {
-				galleryObj.getNext( nextpage, $( this ).parent().parent().parent().parent().parent(), $( this ).parent().parent() );
-			}
+			galleryObj.getNext( nextpage, $( this ).parents( '#item-body' ), $( this ).parents( '.rtm-pagination' ) );
```

Comment:
if possible can we use `closest` instead of `parents` it has better performance.
https://jsperf.com/jquery-parents-vs-closest/2